plugins/lookup/onepassword_ssh_key.py

```diff
@@ -0,0 +1,139 @@
+    OnePass,
+    OnePassCLIv2,
+)
+from ansible.errors import AnsibleLookupError
+from ansible.module_utils.common.text.converters import to_bytes
+from ansible.plugins.lookup import LookupBase
+
+
+class OnePassCLIv2SSHKey(OnePassCLIv2):
+
+    def _get_raw(self, item_id, vault=None, token=None):
+        args = ["item", "get", item_id, "--format", "json"]
+        if vault is not None:
+            args = [*args, f"--vault={vault}"]
+
+        if self.service_account_token:
+            if vault is None:
+                raise AnsibleLookupError(
+                    "'vault' is required with 'service_account_token'"
+                )
+
+            environment_update = {
+                "OP_SERVICE_ACCOUNT_TOKEN": self.service_account_token
+            }
+            return self._run(args, environment_update=environment_update)
+
+        if token is not None:
+            args = [*args, to_bytes("--session=") + token]
+
+        return self._run(args)
+
+    def get_ssh_key(self, item_id, vault=None, token=None, ssh_format=False):
+        _, out, _ = self._get_raw(item_id, vault, token)
+
+        data = json.loads(out)
+
+        if data.get("category") != "SSH_KEY":
+            raise AnsibleLookupError(f"Item {item_id} is not SSH Key")
+
+        for field in data.get("fields", {}):
+            if field.get("id") == "private_key" and field.get("type") == "SSHKEY":
+                return (
+                    field.get("ssh_formats", {}).get("openssh", {}).get("value", "")
+                    if ssh_format
+                    else field.get("value", "")
+                )
+
+
+class LookupModule(LookupBase):
+    def run(self, terms, variables=None, **kwargs):
+        self.set_options(var_options=variables, direct=kwargs)
+
+        ssh_format = kwargs.get("ssh_format")
+        vault = self.get_option("vault")
+        subdomain = self.get_option("subdomain")
+        domain = self.get_option("domain", "1password.com")
+        username = self.get_option("username")
+        secret_key = self.get_option("secret_key")
+        master_password = self.get_option("master_password")
+        service_account_token = self.get_option("service_account_token")
+        account_id = self.get_option("account_id")
+        connect_host = self.get_option("connect_host")
+        connect_token = self.get_option("connect_token")
+
+        op = OnePass(
+            subdomain=subdomain,
+            domain=domain,
+            username=username,
+            secret_key=secret_key,
+            master_password=master_password,
+            service_account_token=service_account_token,
+            account_id=account_id,
+            connect_host=connect_host,
+            connect_token=connect_token,
+            cli_class=OnePassCLIv2SSHKey,
+        )
+        op.assert_logged_in()
+
+        values = []
+        for term in terms:
+            values.append(op._cli.get_ssh_key(term, vault, ssh_format=ssh_format))
+
+        return [values]
```

Comment:
This will return a list, containing one element, which is also a list. Is that intentional? I sampled other plugins, they don't seem to be doing that.